bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/client/AbstractKNXClient.java

```diff
@@ -299,6 +300,11 @@ private void readNextQueuedDatapoint() {
                     logger.warn("Giving up reading datapoint {}, the number of maximum retries ({}) is reached.",
                             datapoint.getDatapoint().getMainAddress(), datapoint.getLimit());
                 }
+            } catch (KnxRuntimeException e) {
+                // KnxRuntimeException is _not_ a subclass of KnxException. Fail gracefully for this case as well,
+                // fixes #7239
+                logger.warn("Error reading datapoint {}: {}", datapoint.getDatapoint().getMainAddress(),
+                        e.getMessage());
```

Comment:
Is the WARN level the good level ? As you finally ignore the error, it looks like something no really important...
Is it something you would like the user to be alerted ?
Is there no risk to flood the log file with recurrent logging ?

Reply:
see comment to jlaur below.... maybe info is fine as well.
Yes, because in item is configured in a way which is not supported or something strange happened during runtime.
Don't think so, as this should happen only at the interval of the scheduled read.